src/index.js

```diff
@@ -60,8 +42,63 @@ const createTask = (input) => {
   `;
   lastLi.className = 'clear';
   placeholder.appendChild(lastLi);
-};
+  document.querySelectorAll('.hello').forEach((e) => {
+    e.addEventListener('keyup', (event) => {
+      listOfTasks[event.target.id].description = event.target.value;
+    });
+  });
+  document.querySelectorAll('li.hello > img').forEach((task) => {
+    task.addEventListener('click', (event) => {
+      if (event.target.src === Trash) {
+        // eslint-disable-next-line no-use-before-define
```

Comment:
- [ ] Hi, your code is working fine but kindly try to fix this ESlint error. this is usually a warning you have written some code that is not clean. please address this issue.